community/server/src/docs/ops/server-configuration.asciidoc

```diff
@@ -146,14 +146,19 @@ Failure to transmit both `X-Forwarded-Host` and `X-Forwarded-Proto` headers will
 
 == Enabling logging from the garbage collector ==
 
-To get garbage collection logging output you have to pass the corresponding option to the server JVM executable by setting the following value in _conf/neo4j-wrapper.conf_:
+To get garbage collection logging output you have to enable GC logging by setting the following value in _conf/neo4j.conf_:
```

Comment:
Policy is to refer to the config file by name only, not path. Elsewhere on the page there should be a link to the "file locations" page to help people find the file.